Feat: Konnect Explorer and summary dashboard documentation (#6961)
Kong/docs.konghq.com: app/konnect/analytics/dashboard.md

---
title: Analytics dashboard
---

![service graph](/assets/images/products/konnect/analytics/konnect-summary-dashboard.png){:.image-border}

You can access a summary dashboard by navigating to [**Analytics** > **Summary**](https://cloud.konghq.com/analytics). The summary dashboard shows performance and health statistics of all your APIs across your organization on a single page and provides greater insights into your service usage.

> From {% konnect_icon analytics %} Analytics, you can view dashboards and access historical data for a range greater than 30 days. You can view the Analytics summary dashboard to track traffic, errors by error code, and latency across all services in your organization.

Reviewer: ```suggestion
From {% konnect_icon analytics %} Analytics, you can view dashboards and access historical data for a range greater than 30 days. You can view the Analytics summary dashboard to track traffic, error rate, and latency across all services in your organization.
```

These categories measure trends by comparing metrics across fixed comparable time intervals and plotting the data points. For example, hour-over-hour, day-over-day, week-over-week, and month-over-month.
You can view a graph for the **Requests**, **Error Rate**, and **P99 Latency** categories:

> * **Requests**: This graph displays the total number of HTTP requests categorized by successful and failed requests over the specified time interval.

Reviewer: We should change this by:
* Remove the 3 bullet points.
* Change the text to something like "This number displays the total number of HTTP requests over the specified time interval. This number also includes any requests made to incorrect routes.

    * Successful requests contain all requests that returned a **1xx-3xx** status code.
    * Failed requests contain all requests that returned a **4xx-5xx** status code.
    * 6xx status codes are not reported.

> * **Error Rate**: This graph displays the total number of failed HTTP requests categorized by error response codes over the specified time interval. Error response codes include any **4xx-5xx** status codes. This graph also includes any requests made to incorrect routes.

Reviewer: ```suggestion
* **Error Rate**: This number displays the total number of failed HTTP requests over the specified time interval. Error response codes include any **4xx-5xx** status codes. This number also includes any requests made to incorrect routes.
```

> * **P99 Latency**: This graph displays request latency, in milliseconds, of the 99th percentile.

Reviewer: ```suggestion
* **P99 Latency**: This number displays request latency, in milliseconds, of the 99th percentile.
```

---
title: Analytics dashboard
---

> ![service graph](/assets/images/products/konnect/analytics/konnect-summary-dashboard.png){:.image-border}

Reviewer: Update image with the current status. I can send you one if required.

Reviewer: @sichvoge to provide

You can access a summary dashboard by navigating to **Analytics** > **Summary**. The summary dashboard shows performance and health statistics of all your APIs across your organization on a single page.
For greater insights into your service usage, access the dedicated {% konnect_icon analytics %} [Analytics](https://cloud.konghq.com/analytics) page.

> From {% konnect_icon analytics %} Analytics, you can view dashboards, access historical data for a range greater than 30 days, and customize the entities in a report:

Reviewer: From here on, we only have the old content that is not correct anymore. Can we update this to just talk about what you can see today in the new dashboard?

    ![errors analytics graph](/assets/images/products/konnect/analytics/konnect-vitals-errors.png){:.image-border}
    > _**Figure 3:** Graph showing errors by 4xx and 5xx error codes received over the past three hours._

* **Latency**: This graph displays request latency, in milliseconds, of the 99th, 95th, and 50th percentiles.
Admins can monitor the latency, investigate where delays are noticed, and optimize performance for APIs.

    {:.note}
    > **Note**: Latency data is only available for requests proxied through a data plane running {{site.base_gateway}} 3.0.0.0 or later.

   ![latency analytics graph](/assets/images/products/konnect/analytics/konnect-analytics-latency.png){:.image-border}
  > _**Figure 4:** Graph showing latency as a percentage over the past 15 minutes._

Graphs can be interacted with, including hovering over chart items to display more details, and filtering options by clicking on items in the **legend**.

You can select a time period using the **time period** drop-down menu. The intervals aggregate data at different increments of time.

> ## Troubleshooting

Reviewer: This section should be at the same level as Explorer, Dashboards, Custom Reports, ...
